luatest/justrun.lua

```diff
@@ -0,0 +1,164 @@
+local fun = require('fun')
+local json = require('json')
+local fiber = require('fiber')
+
+local log = require('luatest.log')
+
+local has_popen, popen = pcall(require, 'popen')
+if not has_popen then
+    log.error('popen module requires Tarantool 2.4.1 and newer')
```

Comment:
Shouldn't we explicitly break here (not just to log) if Tarantool is lower than 2.4.1?

Reply:
As a concise alternative I propose to move the require (without pcall) to `justrun.tarantool`.

If it is called on a supported version, it works. If it is called on an unsupported version, it raises an error that `popen` is not found.

Reply:
I've moved it inside the function :rocket: